Translations update from Weblate

ubports-weblate opened the pull request:
Translations update from [Weblate](https://translate.ubports.com/projects/ubports/lomiri/) for UBports/Lomiri.

dobey commented on po/he.po:
```diff
@@ -457,14 +455,14 @@ msgstr "שום דבר לא מתנגן"
-msgstr[0] "%1 שעה"
-msgstr[1] "%1 שעות"
+msgstr[0] "שעה"
+msgstr[1] "שעתיים"
```
This breaks the translation by removing the argument.

dobey commented on po/he.po:
```diff
@@ -457,14 +455,14 @@ msgstr "שום דבר לא מתנגן"
 msgid "%1 minute"
 msgid_plural "%1 minutes"
-msgstr[0] "דקה %1"
+msgstr[0] "דקה"
```
Breaks translation.

weblate committed:
Translated using Weblate (Hebrew) - 100.0% (185 of 185 strings)

Translated using Weblate (Hungarian) - 100.0% (185 of 185 strings)

Translated using Weblate (Hungarian) - 98.9% (183 of 185 strings)

Translated using Weblate (Hungarian) - 98.9% (183 of 185 strings)

Translated using Weblate (Hungarian) - 98.9% (183 of 185 strings)

Translated using Weblate (Italian) - 100.0% (185 of 185 strings)

Translated using Weblate (Czech) - 100.0% (185 of 185 strings)

Translated using Weblate (Sardinian) - 100.0% (185 of 185 strings)

Translated using Weblate (Portuguese) - 100.0% (185 of 185 strings)

Translated using Weblate (Hebrew) - 100.0% (185 of 185 strings)

Translated using Weblate (Swedish) - 100.0% (185 of 185 strings)

Translated using Weblate (Norwegian Bokmål) - 100.0% (185 of 185 strings)

Translated using Weblate (Thai) - 13.5% (25 of 185 strings)

Translated using Weblate (Portuguese) - 100.0% (185 of 185 strings)

Translated using Weblate (Arabic) - 100.0% (185 of 185 strings)

Translated using Weblate (Thai) - 7.5% (14 of 185 strings)

Translated using Weblate (Thai) - 4.3% (8 of 185 strings)

Translated using Weblate (Spanish) - 100.0% (185 of 185 strings)

Translated using Weblate (Hebrew) - 99.4% (184 of 185 strings)

Translated using Weblate (Arabic) - 100.0% (185 of 185 strings)

Translated using Weblate (Hebrew) - 99.4% (184 of 185 strings)
